ui-components/src/helpers/address.tsx

```diff
@@ -40,6 +40,12 @@ export const MultiLineAddress = (props: AddressProps) => {
       )}
       <Markdown children={props.address.street || ""} />
       <br />
+      {props.address.street2 && (
```

Comment:
Address formats usually have street 2 on the same line, I actually just made this change in another PR I have up if you wanna take a look: https://github.com/bloom-housing/bloom/pull/2675/files#diff-850c89614d0470e86af5b2769705e03fad7e3869c92eb5eca4eb4025329b742aR1